app/next.config.js

```diff
@@ -1,21 +1,16 @@
 /** @type {import('next').NextConfig} */
 
+const { i18n } = require('./next-i18next.config')
+
 const nextConfig = {
-  i18n: {
-    // TODO: implement i18n internationalization-- look into nextJS subpath routing
-    locales: ['en-US', 'es-ES'],
-    defaultLocale: 'en-US',
-    localeSubpaths: {
-      es: 'es',
-    },
-  },
+  i18n,
```

Comment:
just wondering if it's necessary to to pull i18n config in separate file-- It's just an object with three key/val pairs 👀 And I guess the amount of separate config files that have been accumulating in app/ has been causing minimal distress to my inner organizer 😅

Reply:
Yah that's fair! I'll merge it in.